flavors/standard-EXASOL-6.2.0/flavor_base/testconfig

```diff
@@ -1,2 +1,2 @@
 generic_language_tests=python3 python java r
-test_folders=python python3/all python3/external_modules java r standard-flavor/all
+test_folders=python python3/all python3/external_modules java r standard-flavor/all standard-flavor/7.0
```

Comment:
Looks strange, why do we need to test-folder 7.0 to the 6.2.0 standard container?